raiden/utils/typing.py

```diff
@@ -131,6 +131,12 @@
     bound='raiden.messages.RequestMonitoring',
 )
 
+
+T_EventNewBalanceProofReceived = TypeVar(
+    'T_EventNewBalanceProofReceived',
+    bound='raiden.transfer.mediated_transfer.events.EventNewBalanceProofReceived',
+)
```

Comment:
Please remove

Reply:
please follow the conversation at https://github.com/raiden-network/raiden/pull/3374#discussion_r253424031 and https://github.com/raiden-network/raiden/pull/3374/commits/69831a8fab12ea38c4ab98ae1122792d7a5d28ae